### PR TITLE
fixed CQ for bitmessageqt.address_dialogs module

### PR DESCRIPTION
**Pull request by coolguy-cell**
pylint fixes
flake8 fixes
pycodestyle fixes

### CODE REVIEW

File: src/bitmessageqt/address_dialogs.py [hunk 1]
```diff
@@ -259,7 +259,7 @@ def __init__(self, parent=None, config=None):
                 try:
                     mailingListName = config.get(
                         self.address, 'mailinglistname')
-                except:
+                except Exception:
```
Comment (g1itch) [inline src/bitmessageqt/address_dialogs.py:262]: mailingListName = config.safeGet(self.address, 'mailinglistname')

Comment (coolguy-cell) [reply src/bitmessageqt/address_dialogs.py:262]: @g1itch you want me to change this as v0.6 says  mailingListName = config.get(self.address, 'mailinglistname') ?

Comment (g1itch) [reply src/bitmessageqt/address_dialogs.py:262]: There is no need for `try: .. except:` here. Replace `get` by `safeGet` - it handles all appropriate exceptions.

Comment (coolguy-cell) [reply src/bitmessageqt/address_dialogs.py:262]: @g1itch I didnt change anything as pycodestyle complaining for it.

src/bitmessageqt/address_dialogs.py:262:17: E722 do not use bare 'except'

Comment (g1itch) [reply src/bitmessageqt/address_dialogs.py:262]: `except Exception` is worse than bare except

Comment (g1itch) [reply src/bitmessageqt/address_dialogs.py:262]: To fix it properly you should find all possible exception types and list them. It's already done in [bmconfigparser](../blob/v0.6/src/bmconfigparser.py#L107).

Comment (coolguy-cell) [reply src/bitmessageqt/address_dialogs.py:262]: @g1itch sure I 'll make the changes accordingly

Comment (coolguy-cell) [reply src/bitmessageqt/address_dialogs.py]: @g1itch fixed

**Pull request merged**